src/share/poudriere/jail.sh

```diff
@@ -1210,6 +1243,9 @@ else
 	GIT_FULLURL=${proto}://${GIT_BASEURL}
 fi
 
+if [ -z "${KERNEL}" ] && [ "${BUILD_PKGBASE}" -eq 1 ]; then
+    err 1 "pkgbase build need a kernel"
```

Comment:
Why?

Reply:
Because make packages create world+kernel packages.
Also the main reason for using poudriere pkgbase is either to create poudriere images or to distribute packages in sync with kernel (like kmods).
If there is a demand for world only packages I'll add support later.